### DISCUSSION TITLE
newbie...:) need some help on see motion videos from ring doorbell

### DISCUSSION THREAD

**Discussion by 2waz2** [dgreif/ring#735] [Q&A] [answered]
hi, i am new to all of it....manged in 2 hours to do it...:) i would love to know where can i see my video's from motion detected...everything is working....i installed hombridge on win10/node js.... installed ring and can see it in my homebridge/accessories...all is on the same home network....
i also add it to home app on my iphone but i can see only live stream.... i want to know how to see the saved motion videos
thanks for you help...:)
![my accessories](https://user-images.githubusercontent.com/88522884/128930795-a4321533-e102-4a80-93a7-3a047ac0dc23.jpg)

**Comment by dgreif** [comment, marked as answer]
Homebridge only supports live streaming and snapshot.  It does not support viewing recorded video clips.  You will need to use the Ring app for that functionality.

**Reply by 2waz2** [reply]
how can i do snapshot every one minute and where can i see the snapshot?
can i make short video once video motion is detected?
thanks

**Reply by dgreif** [reply]
You can see snapshots in the Home app.  All videos are recorded by Ring and accessible in the Ring app.  You will need a Ring subscription to do recording

**Reply by 2waz2** [reply]
hi i don't see the snap shots in home
does it have history of stored pictures?
where can i find it in home app? thanks

**Reply by dgreif** [reply]
You cannot see a history of snapshots in the Home app. Only the most recent snapshot is available. If you want anything historical, you must use the Ring app
